### ISSUE TITLE
Cosmetic changes to the general Frequency tables main dialog

### ISSUE THREAD

**Issue by rdstern** [IDEMSInternational/R-Instat#8472]
@lilyclements Here is the default dialog completed for a simple 2-way frequency table.

<img width="365" alt="image" src="https://github.com/africanmathsinitiative/R-Instat/assets/11226469/15e3c9c3-9e66-4ba5-9de7-7a6d3624b20b">

And here are the current results:

<img width="252" alt="image" src="https://github.com/africanmathsinitiative/R-Instat/assets/11226469/37e4f415-c5c2-46f9-af67-acf7485972ef">

Here are my suggestions, to make life easier for the user and to get clearer results.
a) @lilyclements do we need the Display box at all?  What's the point of the Summary Variable being the columns?  If we keep it, then I suggest we make the last factor in the list the columns always.  We are producing a table!
b) If we get rid of the Display box then we move the Significant figures control up.  Do we want to make that more general - perhaps then keep the display box, so we could alternatively contol the decimals in the display?
c) The results give NA for the missing counts.  Could we have options - perhaps we do need the display box - perhaps the default could be blank, but zero could be another option.  I think I'd like blank as the default?
d) Could the Margins be ticked by default.
e) When you click Percentages  - detail, small o for Of.  But could the last factor in the list be included as the default.  So, it would start with Variety in the example above.  It isn't often you want cell percentages, so let's make the default a commonly needed option.  

After @lilyclements has checked and possibly provided code then I wonder who could be given this. It should be simple.   @N-thony perhaps to discuss with Lily.  I am really keen we continue to progress on tables for the August release.

**Comment by lilyclements** [comment]
> a) @lilyclements do we need the Display box at all? What's the point of the Summary Variable being the columns? If we keep it, then I suggest we make the last factor in the list the columns always. We are producing a table!

@rdstern you're right that we don't need this display box for the "Frequency" option since we don't have a column to summarise here. I think we want it for the Summaries option though.

> c) The results give NA for the missing counts. Could we have options - perhaps we do need the display box - perhaps the default could be blank, but zero could be another option. I think I'd like blank as the default?

@rdstern do you mean by this an option for what to display NAs as? I can code that in the R code

**Comment by rdstern** [comment]
@Vitalis95 could you start with this issue, (which should be very quick) and then come back to #7999, which is very messy and needs some of the same changes as here.  
The only changes needed for now are as follows:
a) Move the Significant figures out of the display group.
b) Delete the Display box for the Frequency button .  We still need it for the summary tables though.
c) The default lay out should not be a column.  Instead the first factor in the top box becomes the columns. (So it is like the current option below.)
d) When you click Percentages - detail, make small o for Of. And include the last factor in the multiple receiver  as the default. So, it would start with Variety in the example below. It isn't often you want cell percentages, so let's make the default a commonly needed option.

<img width="361" alt="image" src="https://github.com/africanmathsinitiative/R-Instat/assets/11226469/ec7a660d-625a-4eaf-abda-bf637bb13d21">

**Comment by Vitalis95** [comment]
> @Vitalis95 could you start with this issue, (which should be very quick) and then come back to #7999, which is very messy and needs some of the same changes as here. The only changes needed for now are as follows: a) Move the Significant figures out of the display group. b) Delete the Display box for the Frequency button . We still need it for the summary tables though. c) The default lay out should not be a column. Instead the first factor in the top box becomes the columns. (So it is like the current option below.) d) When you click Percentages - detail, make small o for Of. And include the last factor in the multiple receiver as the default. So, it would start with Variety in the example below. It isn't often you want cell percentages, so let's make the default a commonly needed option.
> 
> <img alt="image" width="361" src="https://user-images.githubusercontent.com/11226469/271263783-ec7a660d-625a-4eaf-abda-bf637bb13d21.png">

@Fidel365 , take a quick look at this?

**Comment by lilyclements** [comment]
@rdstern @Vitalis95 if we want only the first variable from the multiple receiver (for now, until Patrick works on the output window to fix that) then is it not confusing to have the multiple receiver up? Why not change this into a single receiver for now?

**Comment by rdstern** [comment]
@lilyclements and @Vitalis95 you are right that it is the 2 variable.  My problem making it into a single receiver is there are 4 different situations, namely numeric by numeric, numeric by factorm etc.  It is only the factor by factor that has the problem, and it isn't also clear when you start, which combination it will be.

**Comment by lilyclements** [comment]
This was fixed by #8562. The remaining issue (c) is being looked at in other issues on displaying NA valeus in tables (#9498)
